web/metrics_controller.go

```diff
@@ -0,0 +1,76 @@
+package web
+
+import (
+	"fmt"
+	"github.com/asdine/storm"
+	"github.com/gin-gonic/gin"
+	"github.com/smartcontractkit/chainlink/services"
+	"github.com/smartcontractkit/chainlink/store/models"
+	"github.com/smartcontractkit/chainlink/utils"
+	"github.com/smartcontractkit/chainlink/web/metric"
+	"net/http"
+	"strings"
+)
+
+// MetricsController fetches high-level information about the nodes usage
+type MetricsController struct {
```

Comment:
Can you clarify the difference between PromController and MetricsController? I get that one is in Prometheus format, and one is not. Do you plan on using both controllers or just one?